File: app/src/main/java/jp/java_conf/ikastyle/Dao/ClothingGearDao.java
package jp.java_conf.ikastyle.Dao;

import androidx.room.Dao;
import androidx.room.Query;

import jp.java_conf.ikastyle.Entity.ClothingGear;

import java.util.List;

@Dao
public interface ClothingGearDao {
    @Query("SELECT * FROM MAST_CLOTHING_GEAR WHERE language_code = :languageCode ORDER BY name")
    public List<ClothingGear> getClothingGearList(int languageCode);
}

File: app/src/main/java/com/splatool/ikastyle/dao/ClothingGearDao.kt
package com.splatool.ikastyle.dao

import com.splatool.ikastyle.entity.ClothingGear
import androidx.room.*

@Dao
interface ClothingGearDao {
    @Query("SELECT * FROM MAST_CLOTHING_GEAR WHERE language_code = :languageCode ORDER BY name")
    fun getClothingGearList(languageCode: Int): List<ClothingGear>
}